src/i_winmusic.c

```diff
@@ -587,14 +595,22 @@ static void FillBuffer(void)
         switch ((int)event->event_type)
         {
             case MIDI_EVENT_META:
-                if (event->data.meta.type == MIDI_META_SET_TEMPO)
+                switch (event->data.meta.type)
                 {
-                    tempo = MAKE_EVT(event->data.meta.data[2],
-                        event->data.meta.data[1], event->data.meta.data[0], 0);
-                    UpdateTempo(delta_time);
-                    break;
+                    case MIDI_META_SET_TEMPO:
+                        tempo = MAKE_EVT(event->data.meta.data[2],
+                            event->data.meta.data[1], event->data.meta.data[0], 0);
+                        UpdateTempo(delta_time);
+                        break;
+
+                    case MIDI_META_END_OF_TRACK:
+                        SendNOPMsg(delta_time);
+                        break;
+
+                    default:
+                        continue;
```

Comment:
Why we ignore `delta_time` for meta events?

Reply:
I think only "end of track" matters for timing. In fact I could set the default case further down to "continue" as well.

Reply:
https://github.com/fabiangreffrath/woof/pull/823/commits/7e71eca6b3d8eec8dfc5a920a40c960c1f0bdc46 covers everything, but I'm not convinced it's needed unless the song has no end of track events.

Reply:
Current version looks good.

So how about copyright? It seems you rewrite half of `i_winmusic.c`. Also not relevant to this PR, but what if I simplify the `switch .. case` a bit in `IsSysExReset()`?

Reply:
> How about adding your copyright to the top of the file?

Is this format okay?
```
// Copyright(C) 2021-2022 Roman Fomin
// Copyright(C) 2022 ceski
```

> Also not relevant to this PR, but what if I simplify the switch .. case a bit in IsSysExReset()?

It's like that to easily add more cases as they're discovered, but I don't mind if you simplify it.

Reply:
> Is this format okay?

Yes, sure.

> It's like that to easily add more cases as they're discovered

Ok, we can keep it as is.